### PR TITLE
#79680 Ensure Linked Posts Attach to Events in CSV Imports

### PR DESCRIPTION
**Pull request by ggwicz**
**Ticket:** [**#79680**](http://central.tri.be/issues/79680)

**Code Review Note:** There are no “props” for this ticket

### CODE REVIEW

File: src/Tribe/Importer/File_Importer_Venues.php [hunk 1]
```diff
@@ -9,23 +9,26 @@ class Tribe__Events__Importer__File_Importer_Venues extends Tribe__Events__Impor
 
 	protected function match_existing_post( array $record ) {
 		$name = $this->get_value_by_key( $record, 'venue_name' );
-		$id   = $this->find_matching_post_id( $name, Tribe__Events__Main::VENUE_POST_TYPE );
+		$id   = $th is->find_matching_post_id( $name, Tribe__Events__Main::VENUE_POST_TYPE );
```
Comment (zbtirrell) [inline src/Tribe/Importer/File_Importer_Venues.php]: this looks like a bug

Comment (ggwicz) [reply src/Tribe/Importer/File_Importer_Venues.php]: it is ! fixed now tho, ready for another review 🔍

File: src/Tribe/Importer/File_Importer_Venues.php [hunk 2]
```diff
@@ -16,16 +16,19 @@ protected function match_existing_post( array $record ) {
 
 	protected function update_post( $post_id, array $record ) {
 		$venue = $this->build_venue_array( $post_id, $record );
+
```
Comment (zbtirrell) [inline src/Tribe/Importer/File_Importer_Venues.php]: the changes in this file seems to be unnecessary code churn

File: src/Tribe/Importer/File_Importer_Organizers.php [hunk 3]
```diff
@@ -16,16 +16,19 @@ protected function match_existing_post( array $record ) {
 
 	protected function update_post( $post_id, array $record ) {
 		$organizer = $this->build_organizer_array( $post_id, $record );
+
```
Comment (zbtirrell) [inline src/Tribe/Importer/File_Importer_Organizers.php]: the changes in this file seems to be unnecessary code churn

**Comment by zbtirrell** [comment]
In general, we should avoid code churn that doesn't add value. That being said, this is approved.

**Pull request merged**